config/flags_service_start.go

```diff
@@ -50,60 +39,46 @@ var (
 		Name:  "agreed-terms-and-conditions",
 		Usage: "Agree with terms & conditions",
 	}
-	// FlagAccessPolicyAddress Trust oracle URL for retrieving access policies.
-	FlagAccessPolicyAddress = cli.StringFlag{
-		Name:  "access-policy.address",
-		Usage: "URL of trust oracle endpoint for retrieving lists of access policies",
-		Value: metadata.DefaultNetwork.AccessPolicyOracleAddress,
-	}
+
 	// FlagAccessPolicyList a comma-separated list of access policies that determines allowed identities to use the service.
 	FlagAccessPolicyList = cli.StringFlag{
 		Name:  "access-policy.list",
 		Usage: "Comma separated list that determines the access policies applied to provide service.",
 		Value: "",
 	}
-	// FlagAccessPolicyFetchInterval policy list fetch interval.
-	FlagAccessPolicyFetchInterval = cli.DurationFlag{
-		Name:  "access-policy.fetch",
-		Usage: `Proposal fetch interval { "30s", "3m", "1h20m30s" }`,
-		Value: 10 * time.Minute,
-	}
-	// FlagShaperEnabled enables bandwidth limitation.
-	FlagShaperEnabled = cli.BoolFlag{
-		Name:  "shaper.enabled",
-		Usage: "Limit service bandwidth",
+
+	// FlagPaymentPricePerGB sets the price per GiB to provided service.
+	FlagPaymentPricePerGB = cli.Float64Flag{
+		Name:  "payment.price-gb",
```

Comment:
We have `wireguard.price-gb`, `openvpn.price-gb`, `noop.price-gb`. 
What is `payment.price-gb`?

Reply:
From the code this flag is used as a fallback, for example if `wireguard.price-gb` is not set `payment.price-gb` will be used.

Reply:
But we already have a default value if the flag not set. Do we need another one?

Reply:
As we do not support "per service" configs (except handy yaml configuring).
UI needs single place to store prices same as it was doing with another "per service" config  - access policy to use for service.

Reply:
UI can set 2 places, and it was doing it with no problem.
I'm just worried that we are adding too much confusion for users with these flags overlap.

Reply:
That's why i recommended to continue with no "per service" support for now as we was living with with with access policies.
Storing to 2 places is workaround which leads to broken UI values after manual yaml edits.